src/scandeval/named_entity_recognition.py

```diff
@@ -145,12 +145,35 @@ def _compute_metrics(
                     labels_no_misc[i][j] = "o"  # type: ignore[call-overload]
 
         # Compute the metrics
-        results = self._metrics["micro_f1"].compute(
-            predictions=predictions, references=labels
+        predictions_all_zero = all(
+            all(ner_tag == "o" for ner_tag in prediction_list)
+            for prediction_list in predictions
         )
-        results_no_misc = self._metrics["micro_f1_no_misc"].compute(
-            predictions=predictions_no_misc, references=labels_no_misc
+        labels_all_zero = all(
+            all(ner_tag == "o" for ner_tag in label_list) for label_list in labels
         )
+        if predictions_all_zero and labels_all_zero:
```

Comment:
Maybe add a comment explaining that its to avoid errors from the micro_f1 metric.